### PR TITLE
Add Bitcoin ABC Vision post

### PR DESCRIPTION
**Pull request by Mengerian**

### CODE REVIEW

File: _posts/2018-08-24-bitcoin-abc-vision.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ 1. Implementing technical measures to enable massive on-chain scaling. The fundamental design of Bitcoin Cash is sound, but there are some technical improvements that must be made to enable Bitcoin Cash to scale to a world-wide currency. We are working to remove immediate bottlenecks such as the mempool admittance code and mining RPC, as well as laying the groundwork to enable massive on-chain scaling well into the future.
+ 2. Improving fungibility. As a cash system, Bitcoin Cash should embrace methods to improve its fungibility.
+ 3. We believe that the basic design of Bitcoin Cash is sound, and does not need any radical change. However, this does not mean it is perfect. It is prudent to make incremental improvements to the system with technically sound design and careful engineering. By implementing optimizations and protocol upgrades, we can enable peer-to-peer digital cash to scale many orders of magnitude beyond current limits.
+
+[Bitcoin ABC Roadmap](https://github.com/Bitcoin-ABC/bitcoinabc.org/blob/master/img/abc-roadmap-whitebg.png)
```
Comment (jasonbcox) [inline _posts/2018-08-24-bitcoin-abc-vision.md]: I think this needs to be prepended by a `!`. and please include the image file as part of this PR

**Comment by Mengerian** [comment]
fixed the issues, and tested locally

**Comment by Mengerian** [comment]
Updated the image path to remove the ".."

**Comment by Mengerian** [comment]
Squash commits

**Pull request merged**